integration_tests/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
+    "experimentalDecorators": true,
```

Comment:
Generally speaking I'm not a fan of using (and in this case being reliant on) "experimental" features.
Do you know how the wider HMPPS dev team feel about using this, and/or what we envisage the risk of TC39 doing something different? 🤔 
https://www.typescriptlang.org/tsconfig/#experimentalDecorators

Reply:
Having said that, it seems quite a few HMPPS projects are using it, so I guess we'd be in good company 😁 
https://github.com/search?q=org%3Aministryofjustice%20experimentalDecorators&type=code

Reply:
They're fairly popular. The risk is probably less that they're unstable and more that they're not official spec so could change later down the road.

Not excusing it, but I think part of the explanation here is simply that JS devs are more comfortable using experimental features since the ecosystem tends to move much faster than official spec.